src/index.html

```diff
@@ -4,9 +4,374 @@
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="catalog">
+        <div class="catalog__filling">
+          <div class="card catalog__card" data-qa="card">
+            <img 
+              class="card__image" 
+              src="./images/imac.jpeg" 
+              alt="iMac image"
+            >
+      
+            <div class="card__info">
+              <p class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+      
+              <p class="card__code">Product code: 195434</p>
+            </div>
+          
+            <div class="card__review-line">
+              <div class="stars stars--4">
+                <span class="stars__star"></span>
+                <span class="stars__star"></span>
+                <span class="stars__star"></span>
+                <span class="stars__star"></span>
+                <span class="stars__star"></span>
+              </div>
+      
+              <p class="card__reviews">Reviews: 5</p>
+            </div>
+      
+            <div class="card__price">
+              <p class="card__price-line">Price:</p>
+              
+              <p class="card__price-value">$2,199</p>
+            </div>
+      
+            <a 
+              class="card__button" 
+              href="/" 
+              data-qa="card-hover"
+              >BUY
+            </a>
+          </div>
+  
+          <div class="card catalog__card">
+            <img 
+              class="card__image" 
+              src="./images/imac.jpeg" 
+              alt="iMac image"
+            >
+      
+            <div class="card__info">
+              <p class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+      
+              <p class="card__code">Product code: 195434</p>
+            </div>
+          
+            <div class="card__review-line">
+              <div class="stars stars--4">
+                <span class="stars__star"></span>
+                <span class="stars__star"></span>
+                <span class="stars__star"></span>
+                <span class="stars__star"></span>
+                <span class="stars__star"></span>
+              </div>
+      
+              <p class="card__reviews">Reviews: 5</p>
+            </div>
+      
+            <div class="card__price">
+              <p class="card__price-line">Price:</p>
+              
+              <p class="card__price-value">$2,199</p>
+            </div>
+      
+            <a 
+              class="card__button" 
+              href="/" 
+              data-qa="hover"
+              >BUY
+            </a>
```

Comment:
```suggestion
            <a 
              class="card__button" 
              href="/" 
              data-qa="hover"
            >
              BUY
            </a>
```
Please fix all cases